bash/lib/e2e.sh

```diff
@@ -114,7 +114,7 @@ function run_web_tests() {
   cd vlayer
 
   bun install --frozen-lockfile
-  SHOULD_DEPLOY_VERIFIER_ROUTER=true bun run test-web:"${VLAYER_ENV}"
```

Comment:
The flag is not doing anything when running `test-web` if I'm not mistaken.

Reply:
It doesn't have any effect on flows I have taken a look at, but it does have effect on some E2E web apps tests running locally by design, which have `deploy` inside their `test-web:dev` scripts. Reverted.